api/container/containerv1/clusters.go

```diff
@@ -31,6 +31,20 @@ type ClusterInfo struct {
 	State             string
 	IsPaid            bool
 	WorkerCount       int
+	Vlans             []Vlan
+}
+
+type Vlan struct {
+	ID      string
```

Comment:
How about just Id?

Reply:
In all the cluster structs we follow ID so used the same